packages/adapter-netlify/README.md

```diff
@@ -31,10 +31,15 @@ Then, make sure you have a [netlify.toml](https://docs.netlify.com/configure-bui
 [build]
   command = "npm run build"
   publish = "build"
+
+[build.environment]
+  AWS_LAMBDA_JS_RUNTIME = "nodejs14.x"
```

Comment:
I think it might be nicer to co-locate the documentation with the setting
```suggestion
  # SvelteKit requires Node 14+, but Netlify uses Node 12 by default
  # https://docs.netlify.com/functions/build-with-javascript/?#runtime-settings
  AWS_LAMBDA_JS_RUNTIME = "nodejs14.x"
```